ffs/manager/manager_test.go

```diff
@@ -41,11 +41,32 @@ func TestCreate(t *testing.T) {
 	require.True(t, id.Valid())
 }
 
+func TestCreateWithFFSMasterAddr(t *testing.T) {
```

Comment:
This seems like a sufficient test. Create the manager with the `ffsUseMasterAddr = true`, create an FFS instance, verify that the FFS instance's address list and default storage config contain the master address.

Reply:
Would be good to have some extra test to cover the case of having an error when indicating using the masteraddr for all FFS instances, but WalletManager not having one. Quite short one since is checking that errs, but just to cover.